From: <idoenmez@suse.de>

acpica: Do not compile in date or time into binaries

Otherwise the data/time compiled in may cause constant re-compilation
in build service environments

Signed-off-by: Thomas Renninger <trenn@suse.de>
---
 source/compiler/aslcompile.c |    9 ++-------
 source/compiler/aslutils.c   |    4 ++--
 source/include/acapps.h      |    8 ++++----
 3 files changed, 8 insertions(+), 13 deletions(-)

Index: acpica-unix2-20121018/source/compiler/aslcompile.c
===================================================================
--- acpica-unix2-20121018.orig/source/compiler/aslcompile.c
+++ acpica-unix2-20121018/source/compiler/aslcompile.c
@@ -160,8 +160,6 @@ void
 AslCompilerFileHeader (
     UINT32                  FileId)
 {
-    struct tm               *NewTime;
-    time_t                  Aclock;
     char                    *Prefix = "";
 
 
@@ -201,12 +199,9 @@ AslCompilerFileHeader (
 
     /* Compilation header with timestamp */
 
-    (void) time (&Aclock);
-    NewTime = localtime (&Aclock);
-
     FlPrintFile (FileId,
-        "%sCompilation of \"%s\" - %s%s\n",
-        Prefix, Gbl_Files[ASL_FILE_INPUT].Filename, asctime (NewTime),
+        "%sCompilation of \"%s\"\n",
+        Prefix, Gbl_Files[ASL_FILE_INPUT].Filename,
         Prefix);
 
     switch (FileId)
Index: acpica-unix2-20121018/source/compiler/aslutils.c
===================================================================
--- acpica-unix2-20121018.orig/source/compiler/aslutils.c
+++ acpica-unix2-20121018/source/compiler/aslutils.c
@@ -454,8 +454,8 @@ UtDisplaySummary (
     {
         /* Compiler name and version number */
 
-        FlPrintFile (FileId, "%s version %X%s [%s]\n\n",
-            ASL_COMPILER_NAME, (UINT32) ACPI_CA_VERSION, ACPI_WIDTH, __DATE__);
+        FlPrintFile (FileId, "%s version %X%s\n\n",
+            ASL_COMPILER_NAME, (UINT32) ACPI_CA_VERSION, ACPI_WIDTH);
     }
 
     /* Summary of main input and output files */
Index: acpica-unix2-20121018/source/include/acapps.h
===================================================================
--- acpica-unix2-20121018.orig/source/include/acapps.h
+++ acpica-unix2-20121018/source/include/acapps.h
@@ -69,15 +69,15 @@
 /* Macros for signons and file headers */
 
 #define ACPI_COMMON_SIGNON(UtilityName) \
-    "\n%s\n%s version %8.8X%s [%s]\n%s\n\n", \
+    "\n%s\n%s version %8.8X%s\n%s\n\n", \
     ACPICA_NAME, \
-    UtilityName, ((UINT32) ACPI_CA_VERSION), ACPI_WIDTH, __DATE__, \
+    UtilityName, ((UINT32) ACPI_CA_VERSION), ACPI_WIDTH, \
     ACPICA_COPYRIGHT
 
 #define ACPI_COMMON_HEADER(UtilityName, Prefix) \
-    "%s%s\n%s%s version %8.8X%s [%s]\n%s%s\n%s\n", \
+    "%s%s\n%s%s version %8.8X%s\n%s%s\n%s\n", \
     Prefix, ACPICA_NAME, \
-    Prefix, UtilityName, ((UINT32) ACPI_CA_VERSION), ACPI_WIDTH, __DATE__, \
+    Prefix, UtilityName, ((UINT32) ACPI_CA_VERSION), ACPI_WIDTH, \
     Prefix, ACPICA_COPYRIGHT, \
     Prefix
 
